subgraphs/compound-forks/aurigami/src/mapping.ts

```diff
@@ -97,13 +97,13 @@ export function handleMarketListed(event: MarketListed): void {
     new MarketListedData(
       protocol,
       new TokenData(
-        underlyingTokenAddr,
+        cTokenAddr,
```

Comment:
Since `MarketListedData` has underlying token before the cToken we should leave it that way.

To me it looks like the issue is that the `cTokenContract` is called for the `underlyingTokenAddr` and vise versa.

Reply:
I now have underlying come before ctoken & fixed the few I missed the first time.